packages/artifact/src/internal/download-http-client.ts

```diff
@@ -264,6 +264,12 @@ export class DownloadHttpClient {
         const gunzip = zlib.createGunzip()
         response.message
           .pipe(gunzip)
+          .on('error', error => {
```

Comment:
The `on('error', ...)` callback on line line 277 isn't called when gunzip fails. I don't think `pipe` propagates errors.

Without this block the error is unhandled, and the new test times out.